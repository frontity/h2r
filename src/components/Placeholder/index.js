import styled from 'styled-components';

export default styled.span.attrs({
  className: 'h2r-lazy-placeholder',
})`
  position: absolute;
  top: 0;
  left: 0;
  box-sizing: border-box;
  width: 100%;
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
`;
