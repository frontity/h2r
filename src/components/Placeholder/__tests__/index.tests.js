import React from 'react';
import renderer from 'react-test-renderer';
import Placeholder from '..';

describe('Placeholder', () => {
  test('has a class named "lazy-placeholder"', () => {
    const element = renderer.create(<Placeholder />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
