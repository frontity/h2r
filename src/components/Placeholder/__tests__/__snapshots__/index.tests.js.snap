// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder has a class named "lazy-placeholder" 1`] = `
<span
  className="sc-bdVaJa h2r-lazy-placeholder kfngWY"
/>
`;
